src/main/kotlin/org/rust/lang/core/resolve/ref/RsModReferenceImpl.kt

```diff
@@ -24,4 +24,6 @@ class RsModReferenceImpl(
 
     override fun isReferenceTo(element: PsiElement): Boolean =
         element is RsFile && super.isReferenceTo(element)
+
+    override fun bindToElement(element: PsiElement): PsiElement? = null
```

Comment:
Why does this method return null?
I can imagine a situation when it can be rebound:
```rust
//- main.rs
mod foo;
mod bar;

//- foo/mod.rs
mod xxx;

//- foo/xxx.rs
// some code

//- bar/mod.rs
mod xxx;
```

And we move `foo/xxx.rs` to `bar` directory

Reply:
Also, there is a magic interface `BindablePsiReference` that we probably should implement.
It's used somewhere in rename refactoring